packages/api/src/controllers/task.ts

```diff
@@ -94,6 +95,29 @@ export function taskParamsWithoutCredentials(
   return result;
 }
 
+export async function getProcessingTasksByRequesterId(
```

Comment:
I meant `task-table.ts`, not the task controller! The tables do the queries, the controllers handle APIs. No need to open a PR to fix this though, commenting for the future.